runtimes/common/src/constants.rs

```diff
@@ -256,9 +256,9 @@ pub mod staking {
 	pub const DEFAULT_BLOCKS_PER_ROUND: BlockNumber = 2 * HOURS;
 
 	#[cfg(feature = "fast-gov")]
-	pub const STAKE_DURATION: BlockNumber = 30;
+	pub const STAKE_DURATION: BlockNumber = 1;
 	#[cfg(not(feature = "fast-gov"))]
-	pub const STAKE_DURATION: BlockNumber = 7 * DAYS;
+	pub const STAKE_DURATION: BlockNumber = 1;
```

Comment:
There's no need to specify features here because the value is the same in both cases.